src/Umbraco.Web.BackOffice/ModelBinders/FromJsonPathAttribute.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Umbraco.Extensions;
+
+namespace Umbraco.Web.BackOffice.ModelBinders
+{
+    /// <summary>
+    ///     Used to bind a value from an inner json property
+    /// </summary>
+    /// <remarks>
+    ///     An example would be if you had json like:
+    ///     { ids: [1,2,3,4] }
+    ///     And you had an action like: GetByIds(int[] ids, UmbracoEntityTypes type)
+    ///     The ids array will not bind because the object being sent up is an object and not an array so the
+    ///     normal json formatter will not figure this out.
+    ///     This would also let you bind sub levels of the JSON being sent up too if you wanted with any jsonpath
+    /// </remarks>
+    internal class FromJsonPathAttribute : ModelBinderAttribute
```

Comment:
Is there a reason to make this internal?  This one seems like a nice, general-purpose attribute that could be reused by other packages.

Reply:
Not really, other then we don't have to maintain that functionality public. I change to public for now, but the reason it was internal, was because the original file was internal `src/Umbraco.Web/Editors/FromJsonPathAttribute.cs`